diff --git a/assets/js/app.js b/assets/js/app.js
new file mode 100644
index 0000000..70f7d36
--- /dev/null
+++ b/assets/js/app.js
@@ -0,0 +1,3 @@
+import cow_say from './cow_say';
+
+console.log(cow_say('Moooo'));
diff --git a/assets/js/cow_say.js b/assets/js/cow_say.js
new file mode 100644
index 0000000..311a13c
--- /dev/null
+++ b/assets/js/cow_say.js
@@ -0,0 +1,4 @@
+// assets/js/cow_say.js
+export default function (msg) {
+    return `The cow says: ${msg}`;
+}
