diff --git a/assets/js/checkout.js b/assets/js/checkout.js
index 352d052..9b804f5 100644
--- a/assets/js/checkout.js
+++ b/assets/js/checkout.js
@@ -1,5 +1,11 @@
+// assets/js/checkout.js
 import '../css/checkout.css'
 import $ from 'jquery';
 import React from 'react';
 
 // ...
+
+
+Encore
+    .enableVersioning()
+    .
\ No newline at end of file
diff --git a/webpack.config.js b/webpack.config.js
index 7ec4760..f54c3c9 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -12,6 +12,7 @@ Encore
     .enableSourceMaps(!Encore.isProduction())
     .disableSingleRuntimeChunk()
     .splitEntryChunks()
+    .enableVersioning()
 ;
 
 module.exports = Encore.getWebpackConfig(); 
